bigquery/dump/load-data-by-POST.py

```diff
@@ -0,0 +1,104 @@
+import sys
```

Comment:
I was using dump as a place to put files that were included in docs but I hadn't had time to clean up yet. I can delete it.